apps/api/migrations/091_webhook_subscriptions_and_deliveries.up.sql

```
@@ -0,0 +1,43 @@
+    -- secret_key_version records which key version sealed it, so key
+    -- rotation can still decrypt older subscriptions' secrets.
+    ADD COLUMN IF NOT EXISTS secret_key_version TEXT,
+    ADD COLUMN IF NOT EXISTS status TEXT NOT NULL DEFAULT 'active'
+        CHECK (status IN ('active', 'suspended')),
+    -- Counts consecutive dead-lettered deliveries (reset to 0 on any
+    -- success). Crossing webhookDeadLetterSuspendThreshold auto-suspends the
+    -- subscription and notifies the owner (#836's auto-suspend requirement).
+    ADD COLUMN IF NOT EXISTS consecutive_dead_letters INT NOT NULL DEFAULT 0,
+    ADD COLUMN IF NOT EXISTS suspended_at TIMESTAMPTZ;
+
+CREATE INDEX IF NOT EXISTS idx_webhooks_status ON webhooks (status);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)